README.md

```diff
@@ -135,6 +137,23 @@ cargo doc --open
 
 You can also view the up-to-date documentation [here](https://maplibre.org/maplibre-rs/docs/api/maplibre/).
 
+## Citing
+
+If you wish to cite this project in a scientific publication use the following format:
+
+```bibtex
+@article{maplibre_rs,
+	title        = {maplibre-rs: toward portable map renderers},
+	author       = {Ammann, M. and Drabble, A. and Ingensand, J. and Chapuis, B.},
```

Comment:
should this say "and MapLibre contributors"?

Reply:
We did not add this to the paper's author list. I think that is also fine because the authors of a paper are usually represent individuals.

MapLibre contributors are mentioned in the paper though.